src/components/Person/Person.tsx

```diff
@@ -0,0 +1,59 @@
+import { Person as IPerson } from '../../types';
+import React from 'react';
+import cn from 'classnames';
+import { NavLink, useParams } from 'react-router-dom';
+
+interface Props {
+  person: IPerson;
+}
+
+export const Person: React.FC<Props> = ({ person }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': personSlug === person.slug })}
+    >
+      <td>
+        <NavLink
+          to={`/people/${person.slug}`}
+          className={cn({ 'has-text-danger': person.sex === 'f' })}
+        >
+          {person.name}
+        </NavLink>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+
+      {person.mother ? (
+        <td>
+          <NavLink
+            to={`/people/${person.mother.slug}`}
+            className="has-text-danger"
+          >
+            {person.mother.name}
+          </NavLink>
+        </td>
+      ) : person.motherName ? (
+        <td>{person.motherName}</td>
+      ) : (
+        <td>-</td>
+      )}
```

Comment:
Use destructurization and destructure all values from `person`, and you can improve your solution 
```suggestion
      {mother
          ? (
             <NavLink
               to={`/people/${mother.slug}`}
               className="has-text-danger"
             >
               {mother.name}
            </NavLink>
          )
          : motherName || '-'}
```

Reply:
Not fixed!

Your code snippet uses nested ternary operators, which can make it hard to read and understand.
- Nested ternary operators can become confusing quickly, making it difficult for others to understand the logic at a glance.

Using clearer structures promotes better coding practices and improves collaboration within a team.

Reply:
honestly i didn't want to destructure all those params, it looks too big for me, but ok